scripts/timeseries/importGeojson.js

```diff
@@ -19,22 +25,40 @@ const allModes = [
   'UNKNOWN'
 ]
 
-const N_DOCS = 75
+const arrayHasSameValue = arr => {
+  return uniqWith(arr, isEqual).length === 1
+}
+
+const getRandomValueFromArray = (arr, exclude) => {
+  const randIdx = Math.floor(Math.random() * arr.length)
+  const selectedValue =
+    exclude && isEqual(exclude, arr[randIdx]) && !arrayHasSameValue(arr)
+      ? getRandomValueFromArray(arr, exclude)
+      : arr[randIdx]
+
+  return selectedValue
+}
 
 const createTrips = () => {
   const trips = []
 
-  for (let i = 0; i < N_DOCS; i++) {
-    const randomDoc = GEODOCS[Math.floor(Math.random() * GEODOCS.length)]
+  for (let idx = 0; idx < N_DOCS; idx++) {
```

Comment:
Nothing, it's just that I find it clearer :)

Reply:
Ok, I'm nitpicking, but I'm not fan of modifying existing code just because it matches more the developer preferences, and not stated in the guidelines. 
No big deal here, but just saying for the general case ;)

Reply:
You are right, there is no guideline, I will cancel this change 👍

Reply:
You can keep it, as long as we agree on this, I'm fine :)